iw/filter_gui.py

```diff
@@ -924,8 +926,6 @@ def _active(info):
             # We use a copy here, so as to not modify the original udev info
             # dict as that would break NameCache matching
             data = mpath[0].copy()
-            # This will swap the device name (eg: sda) for the mpath name (eg: mpatha)
-            data["name"] = udev_device_get_multipath_name(mpath[0])
```

Comment:
The first time an iSCSI mpath converts from single disk to an mpath, it won't detect the change because this overwrites the actual device name (sda) so it doesn't know to remove it.  This is an artifact from the earlier code where it used this instead of the udev call to look up the mpath name.